src/app/modules/main/chat_section/item.nim

```diff
@@ -35,6 +35,9 @@ type
     loaderActive: bool
     locked: bool
     requiresPermissions: bool
+    canPostMessages: bool
+    canPostReactions: bool
```

Comment:
I wonder if this partially duplicates this:
https://github.com/status-im/status-desktop/blob/3ef21897fd70a49bb9bbc5fbc94b1012383e45ae/src/app/modules/main/chat_section/item.nim#L39-L40

It's in https://github.com/status-im/status-desktop/pull/13876

Reply:
Hmm, good question. I guess `canPostMessages` is the equivalent of  `viewAndPostPermissionsSatisfied`.

The `canPostReactions` is necessary for my PR and it's definitely different.

I added the `canPostMessages` because it was given by status-go, but we don't use it for now, so maybe it's worth removing to avoid confusion? cc @endulab